cpp/src/slice2java/JavaUtil.cpp

```diff
@@ -483,7 +483,7 @@ namespace
                 //
                 if ((*i) == "java:getset" &&
                     (!dynamic_pointer_cast<ClassDef>(p) && !dynamic_pointer_cast<Struct>(p) &&
-                     !dynamic_pointer_cast<Slice::Exception>(p) && !dynamic_pointer_cast<DataMember>(p)))
+                     !dynamic_pointer_cast<Exception>(p) && !dynamic_pointer_cast<DataMember>(p)))
```

Comment:
I'd rather keep the qualification. We also have Ice::Exception and it's certainly possible to add a "using namespace Ice" in the future.

Reply:
Alright I'll add it back, but just FYI we use `Exception` without qualification all over the Slice compilers.
This was the only place where we qualified it, hence why I removed the qualification (for consistency).